tests/function.rs

```diff
@@ -81,6 +81,20 @@ fn fn_arg_comma_is_removed() {
     assert!(diff.is_empty());
 }
 
+#[test]
+fn fn_arg_last_character_not_removed() {
+    let diff: ApiCompatibilityDiagnostics = parse_quote! {
+        {
+            pub fn a(a: t, b: t, c: t) {}
+        },
+        {
+            pub fn a(a: t, b: t, c: t) {}
+        },
+    };
+
+    assert!(diff.is_empty());
+}
```

Comment:
A better (and perhaps more explicit) way to ensure that the last argument is still handled is to change its type, and assert that a function modification is reported. This way, we're sure we don't remove arguments from all the functions that have one or more argument.